python/pyspark/sql/connect/plan.py

```diff
@@ -1197,6 +1197,7 @@ def plan(self, session: "SparkConnectClient") -> proto.Relation:
         plan.collect_metrics.input.CopyFrom(self._child.plan(session))
         plan.collect_metrics.name = self._name
         plan.collect_metrics.metrics.extend([self.col_to_expr(x, session) for x in self._exprs])
+        plan.collect_metrics.dataframe_id = self._child._plan_id
```

Comment:
does Spark Connect DataFrame also have the unique id? cc @HyukjinKwon @zhengruifeng

Reply:
the `_plan_id` is unique:

https://github.com/apache/spark/blob/5299e5423f439ce702d14666c79f9d42eb446943/python/pyspark/sql/connect/plan.py#L57-L72

Reply:
yes. plan_id is enough functionally.